File: report-service/src/main/java/com/d3/report/ReportMain.java
/*
 * Copyright D3 Ledger, Inc. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */
package com.d3.report;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import springfox.documentation.swagger2.annotations.EnableSwagger2;

@SpringBootApplication
@EnableSwagger2
public class ReportMain {

    public static void main(String[] args) {
        SpringApplication.run(ReportMain.class, args);
    }
}

File: report-service/src/main/kotlin/com/d3/report/ReportMain.kt
@file:JvmName("ReportMain")

package com.d3.report

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import springfox.documentation.swagger2.annotations.EnableSwagger2

@SpringBootApplication
@EnableSwagger2
class Report

fun main(args: Array<String>) {
    SpringApplication.run(Report::class.java, *args)
}
